### ISSUE TITLE
Overly aggressive render culling leads to invisible chunks

### ISSUE THREAD

**Issue by anarch0possum** [sp614x/optifine#3295]
When i loaded up the optifine 1.15.2 preview, the prominent issue was that almost every chunk was invisible, and wouldn't load until it got updated or blocks adjacent to any of those in the chunk. Chunks would also disappear when an unrendered chunk was between me and that chunk. If i were in an unrendered chunk that wasn't adjacent to a rendered chunk, every chunk would be invisible. This strangeness extended to the game not rendering vertical "chunks" above or below rendered chunks. This issue doesn't seem to affect entities or particles, just blocks. [Here's a youtube video showing off the behavior.] (https://www.youtube.com/watch?v=haTnLLVRifI)

**Comment by sp614x** [comment]
Can you post an F3 screenshot?
Is this only happening in superflat worlds?

**Comment by anarch0possum** [comment]
> 
> 
> Can you post an F3 screenshot?
> Is this only happening in superflat worlds?

I don't seem to be able to replicate the issue, even in the same world. I've seen other issue posts with a similar thing happening in normal type worlds though.

**Comment by devereaux98** [comment]
This happened to me in the nether;

After messing with settings, I turned on "Fast Chunks".  Suddenly I noticed that my iron doors no longer had their opening/closing animation, but i could still walk through them.  I tried to tinker with more settings but it made all the blocks invisible.  The only things that were visible were particle effects and mobs (though when looking at a block i could still see the thin black outline)

Thank you for all you do!
